APR-Core/StatusReport.lua

```diff
@@ -0,0 +1,290 @@
+
+	StatusFrame.Section1.Content.Line1.Text:SetFormattedText(statusInfos.aprVersion[1] .. ': |cff' .. APR.HEXColor.green .. '%s|r', statusInfos.aprVersion[2]) --ToDo: Check if Addon outdated
+	StatusFrame.Section1.Content.Line2.Text:SetFormattedText(statusInfos.wowVersion[1] ..': |cff' .. APR.HEXColor.green .. '%s (build %s)|r', statusInfos.wowVersion[2], statusInfos.wowBuildVersion[2])
+	StatusFrame.Section1.Content.Line3.Text:SetFormattedText(statusInfos.clientLanguage[1] .. ': |cff' .. APR.HEXColor.green .. '%s|r', statusInfos.clientLanguage[2])
+
+    --Section 2 Route Info
+	StatusFrame.Section2 = APR:createStatusSection(300, 105, nil, 30, StatusFrame, 'TOP', StatusFrame.Section1, 'BOTTOM', 0)
+	StatusFrame.Section2.Content = APR:createStatusContent(5, 260, StatusFrame.Section2, StatusFrame.Section2.Header)
+	StatusFrame.Section2.Header.Text:SetFormattedText('Current Route|r')
+
+	--Section 3 Character Info
+	StatusFrame.Section3 = APR:createStatusSection(300, 120, nil, 30, StatusFrame, 'TOP', StatusFrame.Section2, 'BOTTOM', 0)
+	StatusFrame.Section3.Content = APR:createStatusContent(6, 260, StatusFrame.Section3, StatusFrame.Section3.Header)
+	StatusFrame.Section3.Header.Text:SetFormattedText('Character|r')
+
+	StatusFrame.Section3.Content.Line1.Text:SetFormattedText(statusInfos.charFaction[1] .. ': |cff' .. APR.HEXColor.green .. '%s|r', statusInfos.charFaction[2])
+	StatusFrame.Section3.Content.Line2.Text:SetFormattedText(statusInfos.charName[1] .. ': |cff' .. APR.HEXColor.green .. '%s|r', statusInfos.charName[2] .. "-" .. statusInfos.charRealm[2])
+	StatusFrame.Section3.Content.Line4.Text:SetFormattedText(statusInfos.charClass[1] .. ': |cff' .. APR.HEXColor.green .. '%s|r', statusInfos.charClass[2]:lower():gsub("^%l", string.upper))
+end
+
+function APR:updateStatusFrame()
+	local StatusFrame = APR.StatusFrame
+	local statusInfos = APR:getStatusReportInfos()
+    local statusColors = APR:getStatusColors()
+
+	StatusFrame.Section1.Content.Line4.Text:SetFormattedText(statusInfos.currentTime[1] .. ': |cff' .. APR.HEXColor.green ..'%s|r', statusInfos.currentTime[2])
+
+	StatusFrame.Section2.Content.Line1.Text:SetFormattedText(statusInfos.currentRoute[1] .. ': |cff' .. statusColors.currentRouteColor ..'%s|r', statusInfos.currentRoute[2])
+	StatusFrame.Section2.Content.Line2.Text:SetFormattedText(statusInfos.currentStep[1] .. ': |cff' .. statusColors.currentStepColor ..'%s|r', statusInfos.currentStep[2])
+	StatusFrame.Section2.Content.Line3.Text:SetFormattedText(statusInfos.currentZone[1] .. ': |cff' .. statusColors.currentZoneColor ..'%s|r', statusInfos.currentZone[2])
+	StatusFrame.Section2.Content.Line4.Text:SetFormattedText(statusInfos.currentContinent[1] .. ': |cff4beb2c%s|r', statusInfos.currentContinent[2])
+	StatusFrame.Section2.Content.Line5.Text:SetFormattedText(statusInfos.currentCoords[1] .. ': |cff' .. statusColors.currentCoordsColor .. '%s|r', statusInfos.currentCoords[2])
```

Comment:
maybe add the world coord in parenthesis after the map coordinate (you can use UnitPosition("player") + APR.coordinate:RoundCoords(x, y, 2) or math.ceil(x*10000)/100